core/src/main/java/org/bitcoinj/base/LegacyAddress.java

```diff
@@ -66,11 +64,15 @@ public class LegacyAddress implements Address {
      *            20-byte hash of pubkey or script
      */
     private LegacyAddress(Network network, boolean p2sh, byte[] hash160) throws AddressFormatException {
-        this.network = normalizeNetwork(checkNotNull(network));
-        this.bytes = checkNotNull(hash160);
+        if (network == null)
+            throw new NullPointerException();
```

Comment:
This could be `Objects.requireNonNull(network)`

Reply:
Or even:
```
this.network = normalizeNetwork(Objects.requireNonNull(network));
```

We could also us a static import for `Objects.requireNonNull`.

Reply:
This has been resolved via #2819.